### PR TITLE
osdocs-238 add console overview

### PR DESCRIPTION
**Pull request by bmcelvee**
https://jira.coreos.com/browse/OSDOCS-238

### CODE REVIEW

File: modules/web-console-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+related parameter `masterPublicURL` in the `webconsole-config` config map
+defined in the `openshift-web-console` namespace. The web console uses
+WebSockets to maintain a persistent connection with the API server and receive
+updated information as soon as it is available.
+
+The configured host names and IP addresses for the web console are whitelisted
+to access the API server safely even when the browser would consider the
+requests to be cross-origin. To access the API server from a web application
+using a different host name, you must whitelist that host name by specifying the
+`--cors-allowed-origins` option on `openshift start` or from the related master
```
Comment (bmcelvee) [inline modules/web-console-overview.adoc]: @ahardin-rh do you know if `openshift start` is in 4.0?

Comment (ahardin-rh) [reply modules/web-console-overview.adoc]: @bmcelvee Good question. I can get a cluster up tomorrow and check.

Comment (ahardin-rh) [reply modules/web-console-overview.adoc]: @bmcelvee I'm not seeing it, so it could be replaced by `oc cluster up` or similar.

**Comment by bmcelvee** [comment]
Merging with the understanding that there will be follow-up edits.

**Pull request merged**